Core/CoreTests/Store/ReactiveStoreTests.swift

```diff
@@ -449,10 +449,10 @@ class ReactiveStoreTests: CoreTestCase {
         )
     }
 
-    private func createOfflineService() -> OfflineService {
+    private func createOfflineService() -> OfflineModeInteractor {
```

Comment:
The function name should follow the updated class name e.g `createOfflineModeInteractor`.